deploy/docker-compose.yml

```diff
@@ -104,35 +137,86 @@ services:
     environment:
       - ALLOW_HTTP=true
       - PORT=9000
+    networks:
+      - harmony_network
+    platform: linux/amd64
 
   redis:
     image: redis:latest
     restart: always
     pull_policy: always
     ports:
       - 6379:6379
+    networks:
+      - harmony_network
 
   hasura:
-    image: hasura/graphql-engine:latest.cli-migrations-v2
+    image: hasura/graphql-engine:v2.11.3.cli-migrations-v2
     restart: always
     pull_policy: always
     ports:
       - 8080:8080
     environment:
-      - HASURA_GRAPHQL_ENABLE_TELEMETRY=false
-      - HASURA_GRAPHQL_MIGRATIONS_SERVER_TIMEOUT=500
-      - HASURA_GRAPHQL_DATABASE_URL=${GRAPHQL_DATABASE_URL}
+      HASURA_GRAPHQL_ENABLE_TELEMETRY: 'false'
+      HASURA_GRAPHQL_MIGRATIONS_SERVER_TIMEOUT: 500
+      HASURA_GRAPHQL_DATABASE_URL: ${GRAPHQL_DATABASE_URL}
     volumes:
       - ${OUTPUT_PATH}/logs:/logs
+    networks:
+      - harmony_network
+
+  postgres:
+    image: postgres:14
+    restart: always
+    environment:
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_DB: ${POSTGRES_DB:-postgres}
+    ports:
+      - "${POSTGRES_HOST_PORT:-5432}:5432"
+    volumes:
+      - ${PG_DATA:-/usr/local/pgsql/data}:/var/lib/postgresql/data
+    networks:
+      - harmony_network
+    profiles:
+      - postgres
+
+  druid:
```

Comment:
@Ideonvn so here's my crazy hack - we can't do `DRUID_HOST=http://host.docker.internal` (that might work on local, but definitely not on production) ; and there's no way to define a virtual service that just routes network to the correct place.

So what I've done is configure nginx to just pass on traffic on 8082 to http://broker.druid_network:8082 and 8081 to http://coordinator.druid_network:8081